src/runners/alert_dispatcher.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+import importlib
+import json
+import datetime
+import uuid
+
+from .config import CLOUDWATCH_METRICS
+from .helpers import db, log
+from .utils import apply_some, json_dumps
+
+
+def log_alerts(ctx, alerts):
```

Comment:
This function never gets called; it's a leftover from when the alert_dispatcher was the alert_handler and created tickets itself; it's dead code now and can be removed.